packages/dds/matrix/src/test/matrix.stress.spec.ts

```diff
@@ -263,13 +271,14 @@ describe("Matrix", () => {
             { numClients: 5, numOps: 200, syncProbability: 0.0, disconnectProbability: 0, seed: 0x2f98736d },
             { numClients: 2, numOps: 200, syncProbability: 0.3, disconnectProbability: 1, seed: 0x84d43a0a },
         ]) {
+            // eslint-disable-next-line max-len
             it(`Stress (numClients=${numClients} numOps=${numOps} syncProbability=${syncProbability} disconnectProbability=${disconnectProbability} seed=0x${seed.toString(16).padStart(8, "0")})`,
                 // Note: Must use 'function' rather than arrow '() => { .. }' in order to set 'this.timeout(..)'
-                async function () {
-                    this.timeout(15000);
+                async function() {
+                    this.timeout(20000);
```

Comment:
Do you need a faster PC?  :-)

Reply:
I start hitting it with coverage builds.  I guess it wasn't ran with coverage before?